lib/domodule.js

```diff
@@ -46,4 +109,47 @@ class Domodule {
   find(selector) {
     return this.el.querySelectorAll(selector);
   }
+
+  findOne(selector) {
+    const found = this.find(selector);
+
+    if (found.length) {
+      return found[0];
+    }
+
+    return null;
+  }
+
+  findByName(name) {
+    return this.els[name];
+  }
+
+  getOption(option) {
+    return this.options[option];
+  }
+
+  // static methods can't access `this` so they go last
+  static getInstance(name) {
+    return Domodule.refs[name];
+  }
+
+  static discover(el = 'body') {
+    const els = document.querySelectorAll(el);
+
+    for (const matched of els) {
+      const foundModules = matched.querySelectorAll('[data-module]');
+
+      for (const module of foundModules) {
```

Comment:
what about calling it `moduleEl`?  was a little confused when I was reading the code below